doc/source/index.rst

```diff
@@ -2,161 +2,199 @@
 
 .. _homepage:
 
-ArviZ: Exploratory analysis of Bayesian models
-==============================================
+Overview
+========
 
-ArviZ is a Python package for exploratory analysis of Bayesian models. Includes functions for posterior analysis, data storage, sample diagnostics, model checking, and comparison.
+ArviZ
```

Comment:
Is "converting the file to markdown" as _straightforward_ as changing the `index.rst` file to `index.md` + changing the file contents to be markdown? If so, I can try that (just wanted to make sure there are not other steps I'm omitting). I think markdown would be friendlier to edit too.

Reply:
It is not that straightforward, but if it would help you then I can do it in the coming days. Make sure you have pushed everything and ping me on slack so I can do it without generating git conflicts or loosing work in the process.